src/components/table/__spec__.js

```diff
@@ -931,6 +930,21 @@ describe('Table', () => {
       expect(parent.className).toEqual('carbon-table foo');
     });
 
+    it('renders a caption tag when a caption prop is given', () => {
+      let wrapper = render(
```

Comment:
Interesting, I'#ve never actually bench-marked them, but I think `shallow` is the fastest from what I've read, that's it's main benefit